src/Web/Sfa.Das.Sas.Shared.Components/ViewComponents/TrainingProvider/SearchSummary/TrainingProviderSearchSummaryViewComponent.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Sfa.Das.Sas.ApplicationServices.Models;
+using Sfa.Das.Sas.Shared.Components.Orchestrators;
+using Sfa.Das.Sas.Shared.Components.ViewComponents.TrainingProvider.Search;
+using System.Threading.Tasks;
+
+namespace Sfa.Das.Sas.Shared.Components.ViewComponents.TrainingProvider.SearchSummary
+{
+    public class TrainingProviderSearchSummaryViewComponent : ViewComponent
+    {
+        private readonly ITrainingProviderOrchestrator _tpOrchestrator;
+        private readonly IApprenticeshipOrchestrator _apprenticeshipOrchestrator;
+
+        public TrainingProviderSearchSummaryViewComponent(ITrainingProviderOrchestrator tpOrchestrator, IApprenticeshipOrchestrator apprenticeshipOrchestrator)
+        {
+            _tpOrchestrator = tpOrchestrator;
+            _apprenticeshipOrchestrator = apprenticeshipOrchestrator;
+        }
+
+        public async Task<IViewComponentResult> InvokeAsync(TrainingProviderSearchViewModel searchQueryModel, bool inline = false)
+        {
+            var apprenticeshipType = _apprenticeshipOrchestrator.GetApprenticeshipType(searchQueryModel.ApprenticeshipId);
```

Comment:
I see we're calling 2 different orchestrators. My view of the orchestrator is that it's a one to one with a controller. The orchestrator just off-loads the request to the Orchestrator (minus anything http related) and deals with calling application logic and mapping to models.

We can add ticket to re-factor if you agree as I think we have it in a couple of places.

Reply:
Actually maybe just needs that logic moving into helper or something.

Reply:
the issue is, these dont map to a controller and used by view components so there is no one to one mapping and it doesnt make sense to have that type of mapping for a view component. i agree though that there is probably a better way of abstracting this out of the view components.